atlasdb-client/src/main/java/com/palantir/atlasdb/persistent/rocksdb/RocksDbPhysicalPersistentStore.java

```diff
@@ -101,23 +92,19 @@ public Long get(StoreNamespace storeNamespace, @Nonnull Long startTs) {
                 Streams.zip(
                         keys.stream(),
                         byteValues.stream(),
-                        (key, value) -> Maps.immutableEntry(key, deserializeValue(key, value))))
+                        Maps::immutableEntry))
                 .filter(Objects::nonNull)
                 .collectToMap();
```

Comment:
I'm a bit worried about maps with byte-array keys. Let's talk through this in person.